summary-visualiser/src/assets/windTunnel.js

```diff
@@ -0,0 +1,97 @@
+import { formatNumber } from "../util.js";
+
+window.createTrendGraph = function (svgId, trendData, meanValue, windowDuration, yUnit) {
+    const margin = { top: 25, right: 20, bottom: 20, left: 60 };
+    const pointWidth = 40; // Width per data point
+    const width = (trendData.length * pointWidth) - margin.left - margin.right;
+    const height = 120 - margin.top - margin.bottom;
+
+    const svg = d3.select(`#${svgId}`)
+        .attr("width", width + margin.left + margin.right)
+        .attr("height", height + margin.top + margin.bottom)
+        .append("g")
+        .attr("transform", `translate(${margin.left},${margin.top})`);
+
+    const maxVal = d3.max(trendData);
+    const range = maxVal;
+    const yMax = maxVal + (range * 0.05);
+
+    const x = d3.scaleLinear()
+        .domain([0, trendData.length - 1])
+        .range([0, width]);
+
+    const y = d3.scaleLinear()
+        .domain([0, yMax])
+        .range([height, 0]);
+
+    // Create line generator
+    const line = d3.line()
```

Comment:
I think line charts were all the same width, not adjusted based on x max.

Reply:
interesting -- I grant that they look odd when they're all different lengths, but I think that it's important here to keep 10sec time units a consistent width and use the width of the whole graph to show the length of the time series. So a longer time series (e.g., 70 sec vs 20 sec) should be a longer graph. Does that make sense, or is this not useful information?

Reply:
We still have that information in the x max. Idk I would do full width for everything, but I'm not sure one is better than another, so I'll defer to you.